### PR TITLE
Update console.md - Fix Note

### PR DESCRIPTION
**Pull request by Rowantrek**
Modified note - Formatting was incorrect and on the Live Page, the section was not showing inside the Note box.

### CODE REVIEW

**Comment by Rowantrek** [comment]
image

See image for current page view.

File: Teams/rooms/console.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The CreateSrsMedia.ps1 script automates the following tasks:
 4. Assemble the needed components on the installation media.
 
 > [!NOTE]
-A specific version of Windows 10 is required, and this version is only available to volume licensing customers.  You can get a copy from the [Volume Licensing Service Center](https://www.microsoft.com/Licensing/servicecenter/).
+> A specific version of Windows 10 is required, and this version is only available to volume licensing customers.  You can get a copy from the [Volume Licensing Service Center](https://www.microsoft.com/Licensing/servicecenter/).
```
Comment (JohanFreelancer9) [inline Teams/rooms/console.md:66]:
```suggestion
> A specific version of Windows 10 is required, and this version is only available to volume licensing customers. You can get a copy from the [Volume Licensing Service Center](https://www.microsoft.com/Licensing/servicecenter/).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Rowantrek, kindly commit the additional suggested change to remove the extra space. Many thanks.

**Comment by Rowantrek** [comment]
Committed @JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**